### PR TITLE
Add new config consistency test for DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED

### PR DESCRIPTION
**Pull request by zacharycmontoya**
## Motivation

We need to test edge cases for the DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED configuration

## Changes

The test case has the following logic:
- Test:
  - Set DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED=false
  - Pass in a headers with an existing 128-bit trace-id
- Expect: trace-id to be 128-bit trace-ids
  - I think we actually should be generating 64-bit trace-ids in this case, so I'm trying to validate the behavior among the tracing libraries

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Review by mtoffl01 — APPROVED** [review]

File: utils/_context/_scenarios/__init__.py [hunk 1]
```diff
@@ -601,13 +608,21 @@ class _Scenarios:
...
             "DD_TRACE_PROPAGATION_EXTRACT_FIRST": "true",
             "DD_LOGS_INJECTION": "true",
-            "DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED": "false",
         },
         appsec_enabled=False,
         doc="",
         scenario_groups=[ScenarioGroup.TRACING_CONFIG],
     )
 
+    tracing_config_nondefault_4 = EndToEndScenario(
```
Comment (cbeauchesne) [inline utils/_context/_scenarios/__init__.py:617]: Can you add this scenario in `.github/workflows/run-end-to-end.yml` ?

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: utils/_context/_scenarios/__init__.py (hunk 1 above)
Comment (zacharycmontoya) [reply utils/_context/_scenarios/__init__.py:617]: Yep, done in ed6c49a2bff8aae42f421dd4b5cd1d784cbb57b1

**Review by cbeauchesne — APPROVED** [review]
